stetho-repl-js/gradle.properties

```diff
@@ -0,0 +1,4 @@
+POM_NAME=Stetho REPL JavaScript module
+POM_ARTIFACT_ID=stetho-repl-js
```

Comment:
Looks like this rename never occurred?  I'm fine with it as-is, but I will probably rename before we cut the release if that's OK.